webpack/assets/javascripts/react_app/components/TemplateGenerator/TemplateGenerator.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Button } from 'patternfly-react';
+
+import { sprintf, translate as __ } from '../../../react_app/common/I18n';
+import AlertBody from '../common/Alert/AlertBody';
+
+const pollingMsg = `
+  Report %s is now being generated, the download will start once it's done.
+  You can come to this site anytime to get the results.
```

Comment:
You can come to this site anytime to get the results.s/You can come to this page later to get the results. The is stored for 24 hours./